app/templates/home/chat.hbs

```diff
@@ -1,18 +1,20 @@
-{{#if fastboot.isFastBoot}}
-  <div class="md:px-8">
-    <div class="loading-container">
-      <img alt={{t "loading"}} src={{random-loading-spinner}} />
+<div class="mt-5 pt-2 px-5">
+  {{#if fastboot.isFastBoot}}
```

Comment:
I think `{{#if fastboot.isFastBoot}}` has to be the parent element instead of the incoming change `<div class="...">`
If you rearrange the template then the chat part of the site renders normally. I'm not sure what the `mt-5 pt-2 px-5` class is needed for

Reply:
```
{{!-- <div class="mt-5 pt-2 px-5"> --}}
  {{#if fastboot.isFastBoot}}
    <div class="md:px-8">
      <div class="loading-container">
        <img alt={{t "loading"}} src={{random-loading-spinner}} />
      </div>
      <div class="loading-container">
        <h1>{{random-loading-message}}</h1>
      </div>
    </div>
  {{else}}
    <NetworkStatus as |status|>
      <DatafruitsChat
        class="flex flex-col flex-grow min-h-0 flex-basis-0 overflow-hidden md:px-8"
        @networkStatus={{status}}
        @authenticate={{fn this.authenticate}}
      />
    </NetworkStatus>
  {{/if}}
{{!-- </div> --}}
```

Reply:
ahh thanks for that! I had to move the `mt-5 pt-2 px-5` classes from the home template to each individual view template which is why it's there - I was probably going a little too quickly and thought that there would be a test to catch any issue like that :sweat_smile: 

should be fixed now - thanks again!